### ISSUE TITLE
Single node server only? ( Router )

### ISSUE THREAD

**Issue by avboy72** [Matthias247/jawampa#42]
Hello,

I was wondering if this solution is a single server solution... I could not find anything talking about how to run a cluster.

Thanks!

-sean

**Comment by Matthias247** [comment]
Yes, it's a single server solution and there is not something like integrated cluster support.  
There is also no standard for WAMP clustering, but afaik crossbar implements (or wants to implement) some kind of clustering functionality.

I'm also not sure whether WAMP is a really suitable for horizontal scaling in general, especially how consistency and partition tolerance in a cluster could be implemented. The protocol is quite stateful (it is required to store all kind of function providers and event publishers/subscribers and synchronize them also between multiple nodes in the cluster).
